db/structure.sql

```diff
@@ -362,6 +348,71 @@ CREATE SEQUENCE public.holiday_schedules_id_seq
 ALTER SEQUENCE public.holiday_schedules_id_seq OWNED BY public.holiday_schedules.id;
 
 
+--
+-- Name: inaccurate_resource_categories; Type: TABLE; Schema: public; Owner: -
+--
+
+CREATE TABLE public.inaccurate_resource_categories (
+    id bigint NOT NULL,
+    inaccurate_resource_id bigint,
+    category_name text,
+    created_at timestamp without time zone NOT NULL,
+    updated_at timestamp without time zone NOT NULL
+);
+
+
+--
+-- Name: inaccurate_resource_categories_id_seq; Type: SEQUENCE; Schema: public; Owner: -
+--
+
+CREATE SEQUENCE public.inaccurate_resource_categories_id_seq
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+--
+-- Name: inaccurate_resource_categories_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: -
+--
+
+ALTER SEQUENCE public.inaccurate_resource_categories_id_seq OWNED BY public.inaccurate_resource_categories.id;
```

Comment:
Looks like you didn't rollback before changing your migrations

Reply:
I'll clean this up.